### PR TITLE
EUI-3797: Missing payments tab

### PR DESCRIPTION
**Pull request by paul-graham**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3797 ✅ `DONE`

### Change description ###
Fix to ensure the contents of the Payment History tab are shown.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by udaydenduluri33 — COMMENTED** [review]
Can we have a Unit test please

File: src/shared/components/palette/complex/ccd-read-fields-filter.pipe.ts [hunk 1]
```diff
@@ -59,6 +59,10 @@ export class ReadFieldsFilterPipe implements PipeTransform {
     if (field.field_type.type === 'Label' && (field.label || '').length > 0) {
       return true;
     }
+    // We also shouldn't ditch CasePaymentHistoryViewer fields.
+    if (field.field_type.type === 'CasePaymentHistoryViewer') {
+      return true;
+    }
```
Comment (phillip-whitaker) [inline src/shared/components/palette/complex/ccd-read-fields-filter.pipe.ts:65]: I wonder whether the "keep fields" could be put into an array to keep them together etc.

const keepFields = ['Label', 'CasePaymentHistoryViewer'] etc...

(just a suggestion) to reduce the if statements.

**Review by phillip-whitaker — DISMISSED** [review]

File: src/shared/components/palette/complex/ccd-read-fields-filter.pipe.ts (hunk 1 above)
Comment (paul-graham) [reply src/shared/components/palette/complex/ccd-read-fields-filter.pipe.ts:65]: The `'Label'` one has an additional check, or else that's precisely what I would have done. 🤷‍♂️

**Comment by paul-graham** [comment]
> Can we have a Unit test please

I've added 7 for various scenarios.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "2.79.4",
+  "version": "2.79.5-hotfix-missing-payments-tab-alpha",
```
Comment (ronaldmansveld) [inline package.json:3]: Don't forget to make this a proper version number before merging into master

**Review by ronaldmansveld — APPROVED** [review]
Approved, but as mentioned don't forget to update the version number before merging.

File: package.json (hunk 2 above)
Comment (paul-graham) [reply package.json:3]: Argh! Good call! That change will wipe out your approval too! One sec...

Comment (ronaldmansveld) [reply package.json]: It still isn't? It's still a pre-release version.

**Review by udaydenduluri33 — APPROVED** [review]
LGTMp

File: package.json (hunk 2 above)
Comment (paul-graham) [reply package.json]: Changed it to `2.79.5`.

**Review by udaydenduluri33 — APPROVED** [review]
LGTMp

**Review by phillip-whitaker — APPROVED** [review]

**Review by ronaldmansveld — APPROVED** [review]

**Pull request merged**